multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py

```diff
@@ -676,15 +673,15 @@ def create_transaction_for_setting_new_uris(self,
                                                 sender: IAddress,
                                                 token_identifier: str,
                                                 token_nonce: int,
-                                                uris: List[str]) -> Transaction:
-        if not uris:
+                                                new_uris: List[str]) -> Transaction:
+        if not new_uris:
```

Comment:
In a comment, you've asked to remove the check. What is best?

Reply:
Here we need the check because you explicitly call `create_transaction_for_setting_new_uris` so it's ok to have it. I'd say it's not mandatory for methods like `create_transaction_for_updating_metadata` because you may not want to update the URIs.